### PR TITLE
Registration the final stand

### PR DESCRIPTION
**Pull request by dstratossong**
New changes in registration: reworked invitation.

### CODE REVIEW

File: app/assets/javascripts/invitation_requests.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  expertiseChecked[expertise] = true;
+
+  // update checkbox
+  toggleCheckbox(checkboxId);
+
+  // populate
+  var image = "";
+  for (var key in expertiseChecked) {
+    if (expertiseChecked[key]) {
+      if (image !== "") {
```
Comment (tlulu) [inline app/assets/javascripts/invitation_requests.js]: this should be !=

File: app/views/authentication/_simple_registration_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="sign-up">
+  <%= form_for User.new, remote: true do |f| %>
+      <div class="form-fields">
+
+        <!--Invitation Code Carry Over-->
+        <% if phase_active? ProjectMeta::INVITATION_PHASES and invitation_code_id %>
+            <%= f.hidden_field :invitation_code_id, value: invitation_code_id %>
```
Comment (tlulu) [inline app/views/authentication/_simple_registration_form.html.erb]: why do we carry this over?

Comment (dstratossong) [inline app/views/authentication/_simple_registration_form.html.erb]: I haven't thought clearly on this, but I thought I wanted to keep the records of who used which registration code. But that kind of breaks down when they want to register with facebook :(

Comment (tlulu) [inline app/views/authentication/_simple_registration_form.html.erb]: It's fine, lets leave it as it is for now

**Pull request merged**